roles/kubernetes-apps/helm/tasks/main.yml

```diff
@@ -39,3 +39,16 @@
 - name: Helm | Set up bash completion
   shell: "umask 022 && {{ bin_dir }}/helm completion bash >/etc/bash_completion.d/helm.sh"
   when: ((helm_container is defined and helm_container.changed) or (helm_task_result is defined and helm_task_result.changed)) and not ansible_os_family in ["CoreOS", "Container Linux by CoreOS"]
+
+- name: Helm | Install charts
```

Comment:
This should be delegated to the first master, or the helm chart installation will run for each master node

Reply:
Good catch. I'm adding the same `when` condition as for `Helm | Apply Helm Manifests (RBAC)` in the latest commit

Reply:
Maybe `run_once: true` could do the trick instead of a when condition?

Reply:
@mirwan previous tasks in that file are using a `when` condition, so I prefer to keep things consistent.